csatar-plugins/csatar/controllers/jsonimport/membershipcards.htm

```diff
@@ -0,0 +1,29 @@
+<div class="callout fade in callout-info no-subheader">
+    <div class="header">
+        <i class="icon-info"></i>
+        <h3>Imports member cards data to RMCSSZ.</h3>
+        <p>This page can only handle "tagsag.Tagkartya.json".</p>
```

Comment:
A szovegek ezen az oldalon nem kellene magyarul legyenek?

Reply:
Ez nem fog benne maradni a kódban, a többi importnál is angolul volt.